src/openrct2/localisation/FormatCodes.h

```diff
@@ -11,7 +11,9 @@
 
 #include "../common.h"
 
-uint32_t format_get_code(const char* token);
+#include <string_view>
+
+uint32_t format_get_code(std::string_view token);
```

Comment:
`const &`? I don't think string views need them, but because you used above, I'm vouching for consistency.

Reply:
@ZehMatt said to make all `string_view` not `const &`.

Reply:
I know, it's just that there are other places in the diff doing const& (not sure if in the latest revision though)